server.go

```diff
@@ -294,6 +309,17 @@ func makeUDPBuffer(size int) func() interface{} {
 func (srv *Server) init() {
 	srv.queue = make(chan *response)
 
+	const workersCount = 100
```

Comment:
I’m very much against this kind of magic tuning variable. It’s impossible to pick a suitable value. Too small and performance is artificially limited. Too high and you’ll still get the OOM and crashes as before. What’s worse is that the correct value isn’t even static and depends on what the system is otherwise doing.